javascript-light-bulb/main.js

```diff
@@ -0,0 +1,16 @@
+var $lightBulb = document.querySelector('.lightbulb');
+var $container = document.querySelector('.container');
+var clickCount = 0;
+
+function click(event) {
+  clickCount++;
```

Comment:
Instead of using a click count, I think you can accomplish this with a variable that holds a boolean value